bears/css/StyleLintBear.py

```diff
@@ -0,0 +1,31 @@
+import os
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='stylelint',
+        output_format='regex',
+        output_regex=r'.+\n\s(?P<line>\d+):(?P<column>\d+)'
+                     r'\s+✖\s(?P<message>.+)')
+class StyleLintBear:
+    """
+    Checks the code with stylelint. This will run stylelint over each file
+    separately.
+
+    Detect errors and potential problems in CSS code and to enforce
+    appropriate coding conventions. For example, problems like syntax errors,
+    invalid color codes etc can be detected.
+
+    For more information on the analysis visit <http://stylelint.io/>
+    """
+    LANGUAGES = {'CSS'}
+    REQUIREMENTS = {PipRequirement('stylelint', '7.7')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
```

Comment:
Um it's good if you include `CAN_DETECT` as well, other than the place where you have mentioned in the comments.